tests/foreman/cli/test_contentview.py

```diff
@@ -2133,9 +2301,58 @@ def test_positive_user_with_all_cv_permissions(self):
         @assert: User is able to perform create, read, modify, promote, publish
         content view
 
-        @caseautomation: notautomated
-
+        @CaseLevel: Integration
         """
+        cv = make_content_view({'organization-id': self.org['id']})
+        password = gen_string('alphanumeric')
+        user = make_user({'password': password})
+        role = make_role()
+        make_filter({
+            'organization-ids': self.org['id'],
+            'permissions': PERMISSIONS['Katello::ContentView'],
+            'role-id': role['id'],
+        })
+        make_filter({
+            'organization-ids': self.org['id'],
+            'permissions': PERMISSIONS['Katello::KTEnvironment'],
+            'role-id': role['id'],
+        })
+        User.add_role({
```

Comment:
Can we check that only these permissions are applied and that user is not an admin user? Otherwise, you will have too many rights there